docs/data/data-grid/layout/layout.md

```diff
@@ -8,6 +8,20 @@ Instead, it takes up the space given by its parent.
 The Data Grid will raise an error in the console if its container has no intrinsic dimensions.
 :::
 
+## Flex parent container
+
+The Data Grid can be placed inside a flex container with `flex-direction: column`.
+Without setting the minimum and maximum height, the Data Grid will take as much space as it needs to display all rows.
+This is effectively equivalent to setting `autoHeight={true}`, but with the added benefit of being able to set the minimum and maximum height of the parent container.
```

Comment:
Given the above, I feel like we should deprecate the `autoHeight` prop now.
What do you think?

Reply:
Sounds good to me.

Reply:
Done 👍🏻

<img width="680" src="https://github.com/user-attachments/assets/f95ef40d-125d-474e-8072-2fac49f5e043">